snpEff/4.3t/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM staphb/bcftools:1.10.2
+MAINTAINER Daniel Miller (millerd15@chop.edu)
+
+ENV SNPEFF_VERSION 4_3t
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt update && apt install -y wget openjdk-8-jdk unzip
```

Comment:
Do you need this '-y' as you have already set DEBIAN_FRONTEND=noninteractive?

Reply:
Probably not but I don't think I've tried it. Either way it doesnt change anything.